src/ctap/mod.rs

```diff
@@ -99,6 +100,11 @@ pub const FIDO2_VERSION_STRING: &str = "FIDO_2_0";
 #[cfg(feature = "with_ctap1")]
 pub const U2F_VERSION_STRING: &str = "U2F_V2";
 
+pub const CREDENTIAL_PARAMETER: PublicKeyCredentialParameter = PublicKeyCredentialParameter {
```

Comment:
I don't think `CREDENTIAL_PARAMETER` is descriptive enough of what this constant contains, which makes it hard to understand in the other contexts. What parameter (public key, ES256)? Why is it fixed? A line of documentation would also be welcome.

Reply:
Renamed to ES256_CRED_PARAM. This also coincides with the specification, where the types are always spelled out like CredentialParameter, and the variables are shortened like cred_param.